src/pages/home.html

```diff
@@ -31,12 +31,16 @@ <h1 class="card-title">{{page.title}}</h1>
         <div class="col-lg-12 home-rdf-resource-search">
             <h3><span  tooltip="View RDF data as a text table or as a visual graph" tooltip-placement="right">View resource</span></h3>
             <div class="card">
-                <div class="card-block">
+                <div class="card-block" ng-keydown="onKeyDown($event)">
```

Comment:
You must define the function to be called in the controller! (homeCtrl or something)

Reply:
If the default behavior is different

Reply:
agree